gcp-cloud-run/src/main/java/ai/h2o/mojos/deploy/gcp/cloud/run/config/EnvironmentConfiguration.java

```diff
@@ -0,0 +1,82 @@
+package ai.h2o.mojos.deploy.gcp.cloud.run.config;
+
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.Storage;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Splitter;
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EnvironmentConfiguration {
+
+  private static final Logger log = LoggerFactory.getLogger(EnvironmentConfiguration.class);
+  private static final String MOJO_DOWNLOAD_PATH = "/tmp/pipeline.mojo";
+  private static final String LICENSE_DOWNLOAD_PATH = "/tmp/license.sig";
+  private Storage storage;
+
+  public EnvironmentConfiguration(Storage storage) {
+    this.storage = storage;
+  }
+
+  /**
+   * Method for ensuring scoring environment is correct and has expected files: pipeline.mojo and
+   * license.sig.
+   */
+  public void configureScoringEnvironment() {
+    try {
+      Map<String, String> env = System.getenv();
+      Preconditions.checkArgument(
+          !env.getOrDefault("DRIVERLESS_AI_LICENSE_FILE", "").isEmpty(),
+          "Environment Variable: 'DRIVERLESS_AI_LICENSE_FILE' must be set");
+      downloadFromGcs(env);
+      File mojoFile = Paths.get(MOJO_DOWNLOAD_PATH).toFile();
+      File licFile = Paths.get(LICENSE_DOWNLOAD_PATH).toFile();
+      Preconditions.checkArgument(mojoFile.exists(), "File: %s, must exist", mojoFile.toString());
```

Comment:
Apologies that I misled you :peace:. I don't think there is value in these two checks. This should never happen right?
I meant checking that the env vars are set, but didn't notice the check in the `getFromEnv` which I guess is sufficient.

Reply:
true true... this should never happen... I figured it was... safer :man_shrugging: to double check, but you are right if gcs client doesn't download the file, it would never exist and throw up.